src/view/SignupView.java

```diff
@@ -25,13 +24,25 @@ public class SignupView extends JPanel implements ActionListener, PropertyChange
   private final JButton cancel;
 
   public SignupView(SignupController controller, SignupViewModel signupViewModel) {
+    this.setBackground(Color.YELLOW);
 
     this.signupController = controller;
-    this.signupViewModel = signupViewModel;
     signupViewModel.addPropertyChangeListener(this);
 
+    JPanel body = new JPanel();
+    body.setBackground(Color.RED);
```

Comment:
This colour is great, although a future PR should probably pull it out into a separate file of UI constants

Reply:
Can you create a new issue for this so we can track it? Otherwise, looks great

Reply:
Added a commit for a TODO, maybe that's enough? (If not I can also create the issue)

Reply:
Yup, that's fine. You're good to merge.